### ISSUE TITLE
HappyPanda v0.6.0 error trying launch

### ISSUE THREAD

**Issue by FinderX** [happypandax/happypandax#80]
I have this error but I dont know what it is.
```
user@mint-pc ~/Programs/happypandax $ ./happypandax_gui
Traceback (most recent call last):
  File "gui.py", line 1, in <module>
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 631, in exec_module
  File "happypanda/common/patch.py", line 2, in <module>
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 631, in exec_module
  File "site-packages/gevent/__init__.py", line 87, in <module>
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 631, in exec_module
  File "site-packages/gevent/_hub_local.py", line 101, in <module>
  File "site-packages/gevent/_util.py", line 105, in import_c_accel
  File "importlib/__init__.py", line 126, in import_module
ModuleNotFoundError: No module named 'gevent.__hub_local'
[5811] Failed to execute script gui
```

**Comment by FinderX** [comment]
I read ` File "/home/travis/virtualenv/python3.6.3/` and I don't have the user "travis".
I think this is hard coded...

**Comment by twiddli** [comment]
facepalm, I can't believe there's no test for this yet, thanks for the report

**Comment by twiddli** [comment]
fixed in `v0.6.1`
